scripts/kibana_proxy.py

```diff
@@ -126,7 +131,8 @@ def print_instructions():
                     assert f.result() is None
         finally:
             for container in containers:
-                container.kill()
+                if container.status == 'running':
```

Comment:
If the container enters a `stopped` state during the duration of the script, then the operator kills the script the container is still alive. 🤔